docs/install/install_everest_helm_charts.md

```diff
@@ -0,0 +1,110 @@
+# Install Percona Everest using Helm
+
+This section explains how to install Percona Everest using [Helm](https://helm.sh/) as an alternative to `everestctl`. Helm charts simplify the deployment process by packaging all necessary resources and configurations, making them ideal for automating and managing installations in Kubernetes environments.  
+
+## Install Percona Everest and deploy namespaces
+
+Here are the steps to install Percona Everest and deploy additional namespaces:
+{.power-number}
+
+1. Add the Percona Helm repository:
+
+    ```sh
+    helm repo add percona https://percona.github.io/percona-helm-charts/
+    ```
+
+2. Install Percona Everest:
+
+    ```sh
+    helm install everest-core percona/everest \
+    --namespace everest-system \
+    --create-namespace
+    ```
+
+    ??? info "What's happening under the hood"
+        The command does the following:
+        {.power-number}
+
+        1. Deploys the Percona Everest components in the `everest-system` namespace. Currently, specifying a different namespace for Percona Everest is not supported.
+
+        2. Deploys a new namespace called `everest` for your databases and the database operators.
+
+            You can override the name of the database namespace by using the `dbNamespace.namespaceOverride` parameter. If you prefer to deploy just the core components, set `dbNamespace.enabled=false`
+
+3. Once the installation is complete, retrieve the `admin` password:
+
+    ```sh
+    kubectl get secret everest-accounts -n everest-system -o jsonpath='{.data.users\.yaml}' | base64 --decode  | yq '.admin.passwordHash'
+    ```
+
+4. Connect to the Percona Everest UI by setting up port-forwarding to your local machine:
+
+    ```sh
+    kubectl port-forward svc/everest -n everest-system 8080:8080
+    ```
```

Comment:
We should also mention the other methods (NodePort, LoadBalancer) of accessing everest as we do in the everestctl installation page.

Reply:
Yes. I am still working on it.